### PR TITLE
Update information about Tibs and their sessions

### PR DESCRIPTION
**Pull request by tibs**
I would have tested in Docker, but it failed to install psycopg2, perhaps because I'm on a Mac (it said `warning: manifest_maker: standard file '-c' not found`).

### CODE REVIEW

File: sessions/talk/a-brief-history-of-markup-languages.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ video:
 ---
 There's a long history to text markup languages, and I don't think most people know much of it.
 
-I hope to touch (briefly) on: linotype, nroff/troff, SGML, HTML, Docbook, TeX and LaTeX, setext, reStructuredText, markdown and asciidoctor.
+I hope to touch (briefly) on at least nroff/troff, SGML, HTML, Docbook, TeX and LaTeX, setext, reStructuredText, markdown and AsciiDoctor.
```
Comment (alexwlchan) [inline sessions/talk/a-brief-history-of-markup-languages.md]: A very minor nit: AsciiDoc is the format, Asciidoctor (no uppercase “d”) is a Ruby library for parsing AsciiDoc.

No, I don’t know why they’re inconsistent either.

 ¯\\\_(ツ)_/¯

**Review by alexwlchan — APPROVED** [review]
One very minor nit that @tibs might want to correct, but otherwise LGTM.

File: sessions/talk/a-brief-history-of-markup-languages.md (hunk 1 above)
Comment (tibs) [reply sessions/talk/a-brief-history-of-markup-languages.md]: Ah - thanks for spotting that. I shall fix it. (and I do know why they're different - but I can explain later)

**Comment by alexwlchan** [comment]
🎉

**Pull request merged**